compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -363,6 +367,17 @@ public BallerinaVariableSymbol createVariableSymbol(BVarSymbol symbol, String na
                 .build();
     }
 
+    /**
+     * Create a symbol for a tuple member.
+     *
+     * @param symbol {@link BVarSymbol} to convert
+     * @return {@link BallerinaTupleMemberSymbol} generated
+     */
+    public BallerinaTupleMemberSymbol createTupleMember(BVarSymbol symbol) {
+        TypeSymbol type = typesFactory.getTypeDescriptor(symbol.getType());
+        return new BallerinaTupleMemberSymbol(context, symbol, type);
```

Comment:
Ditto.

```suggestion
        return new BallerinaTupleMemberSymbol(context, (TupleTypeSymbol) symbol, type);
```

Reply:
This is the member of the tuple-type and the symbol here is a BVarSymbol. We can't cast it to a tuple-type symbol